contrib/fwupd.spec.in

```diff
@@ -315,6 +315,7 @@ mkdir -p --mode=0700 $RPM_BUILD_ROOT%{_localstatedir}/lib/fwupd/gnupg
 %{_libdir}/fwupd-plugins-3/libfu_plugin_altos.so
 %{_libdir}/fwupd-plugins-3/libfu_plugin_amt.so
 %{_libdir}/fwupd-plugins-3/libfu_plugin_ata.so
+%{_libdir}/fwupd-plugins-3/libfu_plugin_ccgx.so
```

Comment:
Is similar functionality used for anything outside of the CCGx family?  If so, maybe it's better to be `libfu_plugin_cypress.so`.  If it's very CCGx specific this is fine.

Reply:
Its specific to CCGx family of devices.